tests/foreman/api/test_webhook.py

```diff
@@ -159,13 +161,14 @@ def test_positive_event_triggered(self, module_org, module_target_sat):
 
         :CaseImportance: Critical
         """
-        hook = module_target_sat.api.Webhooks(
+        hook = target_sat.api.Webhooks(
             event='actions.katello.repository.sync_succeeded', http_method='GET'
         ).create()
-        repo = module_target_sat.api.Repository(
+        repo = target_sat.api.Repository(
             organization=module_org, content_type='yum', url=settings.repos.yum_0.url
         ).create()
-        with module_target_sat.api.session.shell() as shell:
+        with target_sat.api.session.shell() as shell:
```

Comment:
@pondrejk can you please update this to 
```suggestion
        with target_sat.session.shell() as shell:
```
somehow it got typo from my recent change